gazelle/generate.go

```diff
@@ -96,6 +125,47 @@ func defaultModuleName(args language.GenerateArgs) string {
 	return defaultModuleName
 }
 
+func generateRuleFromProtoPackage(protoPackageName string, protoPackage proto.Package) []*rule.Rule {
+	swiftProtoLibraryName := strings.TrimSuffix(protoPackageName, "_proto") + swiftProtoSuffix
```

Comment:
It looks like we use the output of `strings.TrimSuffix(protoPackageName, "_proto")` multiple times in this function. Could you set it to a variable and use it throughout the function?